src/toil/wdl/wdl_functions.py

```diff
@@ -226,7 +225,7 @@ def process_infile(f: Any, fileStore: AbstractFileStore):
     elif isinstance(f, (int, str, bool, float)):
         return f
     else:
-        raise WDLRuntimeError('Error processing file: '.format(str(f)))
+        raise WDLRuntimeError('Error processing file: {0}'.format(str(f)))
 
 
 def sub(input_str: str, pattern: str, replace: str) -> str:
```

Comment:
```suggestion
def sub(input_str: Union[str, WDLFile], pattern: Union[str, WDLFile] replace: Union[str, WDLFile]) -> str:
```

From https://ucsc-ci.com/databiosphere/toil/-/jobs/89507#L1492

Reply:
Why wasn't my suggestion kept? Below we clearly see that all three parameters allowed to be of type `WDLFile`..